File: app/src/main/java/com/antyzero/smoksmog/ui/screen/start/fragment/LocationStationFragmentComponent.java
package com.antyzero.smoksmog.ui.screen.start.fragment;


import com.antyzero.smoksmog.google.GoogleModule;
import com.antyzero.smoksmog.ui.screen.FragmentModule;

import dagger.Subcomponent;

@Subcomponent(
        modules = {
                FragmentModule.class,
                GoogleModule.class
        }
)
public interface LocationStationFragmentComponent {

    void inject(LocationStationFragment fragment);
}

File: app/src/main/kotlin/com/antyzero/smoksmog/ui/screen/start/fragment/LocationStationFragmentComponent.kt
package com.antyzero.smoksmog.ui.screen.start.fragment


import com.antyzero.smoksmog.google.GoogleModule
import com.antyzero.smoksmog.ui.screen.FragmentModule

import dagger.Subcomponent

@Subcomponent(modules = arrayOf(FragmentModule::class, GoogleModule::class))
interface LocationStationFragmentComponent {

    fun inject(fragment: LocationStationFragment)
}
